examples/ecommerce-order-example.php

```diff
@@ -2,7 +2,27 @@
 
 date_default_timezone_set('America/Sao_Paulo');
 
-require __DIR__.'/../vendor/autoload.php';
+// FIX: MOZG
+
+$depth = 3;
+$path = dirname(__FILE__);
+for( $d=1 ; $d <= $depth ; $d++ ){
```

Comment:
Por favor, altere essa linha para seguir o padrão [PSR-2 (5.4 - for)](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md#54-for)